server/src/server.ts

```diff
@@ -7990,7 +7990,7 @@ Email verified! You can close this tab or hit the back button.
       .then(function (zinvite: any) {
         // NOTE: the counter goes in the email body so it doesn't create a new email thread (in Gmail, etc)
 
-        body += createProdModerationUrl(zinvite);
+        body += createModerationUrl(zinvite);
```

Comment:
Ah wait, this needs `req` as a first parameter.

Would it be easy for you to validate visually that this works as expected? My sense is that this was untested, yes?

Reply:
The perils of late-night development. Sorry about this. It does run and the email looks sort of right at first glance, but it's not.

Reply:
Haha no worries, I really appreciate your upstreaming this stuff :)